tests/js/server/replication2/replication2-supervision-replicated-state-target-cluster.js

```diff
@@ -93,14 +91,45 @@ const replicatedStateSuite = function () {
     };
   }());
 
+  const createReplicatedState = function () {
+    const stateId = lh.nextUniqueLogId();
+
+    const servers = _.sampleSize(lh.dbservers, targetConfig.replicationFactor);
+    let participants = {};
+    for (const server of servers) {
+      participants[server] = {};
+    }
+
+    updateReplicatedStateTarget(database, stateId,
+        function () {
+          return {
+            id: stateId,
+            participants: participants,
+            config: targetConfig,
+            properties: {
+              implementation: {
+                type: "black-hole"
+              }
+            }
+          };
+        });
+
+    lh.waitFor(spreds.replicatedStateIsReady(database, stateId, servers));
+    const leader = lh.getReplicatedLogLeaderPlan(database, stateId).leader;
+    const followers = _.difference(servers, [leader]);
+    return {stateId, servers, leader, followers};
+  };
+
```

Comment:
This is the same as in replicated-state-helper. It's good, but I'm making a note to replace it in my upcoming PR.